var/spack/repos/builtin/packages/py-cmake/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCmake(PythonPackage):
+    """CMake is an open-source, cross-platform family of tools designed to
+    build, test and package software
+    """
+
+    homepage = "https://www.example.com"
+    url      = "https://files.pythonhosted.org/packages/bf/b6/bc83a846ac9ffbcb7f9490af135c42002e12f3adc8253c9e55c07de49cf3/cmake-3.18.0.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url

Reply:
Now python hosted links are automatically converted to pypi links right? (or let me just read the docs)